.clang-format

```diff
@@ -48,35 +48,35 @@ BraceWrapping:
     BeforeElse: false
     IndentBraces: false
 
-# When constructor initializers do not fit on one line, put them each on a new line.
+ # When constructor initializers do not fit on one line, put them each on a new line.
 ConstructorInitializerAllOnOneLineOrOnePerLine: true
 # Indent initializers by 4 spaces
 ConstructorInitializerIndentWidth: 4
 
-# Indent width for line continuations.
+ # Indent width for line continuations.
 ContinuationIndentWidth: 8
 
-# No indentation for namespaces.
+ # No indentation for namespaces.
 NamespaceIndentation: None
 
-# Horizontally align arguments after an open bracket.
+ # Horizontally align arguments after an open bracket.
 # The coding style does not specify the following, but this is what gives
 # results closest to the existing code.
 AlignAfterOpenBracket: true
 AlwaysBreakTemplateDeclarations: true
 
-# Ideally we should also allow less short function in a single line, but
+ # Ideally we should also allow less short function in a single line, but
 # clang-format does not handle that.
 AllowShortFunctionsOnASingleLine: Inline
 
-# The coding style specifies some include order categories, but also tells to
+ # The coding style specifies some include order categories, but also tells to
 # separate categories with an empty line. It does not specify the order within
 # the categories. Since the SortInclude feature of clang-format does not
 # re-order includes separated by empty lines, the feature is not used.
 SortIncludes: false
 
-# macros for which the opening brace stays attached.
+ # macros for which the opening brace stays attached.
 ForEachMacros: [foreach, Q_FOREACH, BOOST_FOREACH, forever, Q_FOREVER, QBENCHMARK, QBENCHMARK_ONCE]
 
-# Break constructor initializers before the colon and after the commas.
-BreakConstructorInitializers: BeforeColon
+ # Break constructor initializers before the colon and after the commas.
+BreakConstructorInitializers: BeforeColon
```

Comment:
Please, configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).